KLSadd_insertion/src/updateChapters.py

```diff
@@ -114,48 +371,74 @@ def findReferences(chapter):
     elif chapticker == 1:
         chaptercheck = str(14)
     # canAdd tells the program whether the next section is a reference
-    canAdd = False
+    canadd = False
```

Comment:
This is an example of code which should be self-documenting - instead of having a comment describing what the variable is, name it something that describes it.